### DISCUSSION TITLE
expand wikipedia article

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [free-audio/clap#401] [General]
https://en.wikipedia.org/wiki/CLever_Audio_Plug-in

the wikipedia article is a stub, i feel this needs to be expanded for adoption purposes and clarity for people who are not familiar with clap. music producers typically do not browse github documents to learn about software, or even know what github is

**Comment by baconpaul** [comment]
It should definitely link to the CLAP website: https://cleveraudio.org

**Reply by RustoMCSpit** [reply]
it does

**Reply by baconpaul** [reply]
oh right. the GitHub repository appears before the website so i missed that
